### ISSUE TITLE
Property editors breaking tk.Slider (and tk.IntSlider)

### ISSUE THREAD

**Issue by SugoiDev** [jacobdufault/fullinspector#192]
Very simple to reproduce (extracted directly from my working code):

```
...
new tk.Slider(
    nameof(_paddingAdjustment).FiLabel(),
    -10f,
    10f,
    (o, c) => o._paddingAdjustment,
    (o, c, v) => o._paddingAdjustment = v
),
new tk.PropertyEditor(nameof(_editorEnableGizmos).FiLabel(), nameof(_editorEnableGizmos)),
new tk.Slider(
    nameof(_paddingAdjustment).FiLabel(),
    -10f,
    10f,
    (o, c) => o._paddingAdjustment,
    (o, c, v) => o._paddingAdjustment = v
),
...
```

It'll be rendered like this:

![screenshot 2016 11 15-09 11 17](https://cloud.githubusercontent.com/assets/7355630/20303407/604617ce-ab12-11e6-87c0-97dc44366f8a.png)

Anyone got any idea why that is happening?
I don't see anything strange in tk.Slider or tk.IntSlider, and I'm not knowledgeable enough about FullInspector's generic properties internals to dig deep in there.

**Comment by SugoiDev** [comment]
Some progress: passing fiGUIContent.Empty instead of a label makes is kinda work:

![screenshot 2016 11 15-09 11 27](https://cloud.githubusercontent.com/assets/7355630/20303985/1cb83610-ab15-11e6-887f-caeb7fb09005.png)

**Comment by jacobdufault** [comment]
This should be fixed in dd5b1401e329e3fa3aef96078720e3bc974e0688.
